.gitignore

```diff
@@ -253,3 +249,6 @@ test/*.deletedb
 *.deletedb
 src/cli/*.deletedb
 
+# pyenv python version (for node-gyp builds) (pyenv local 2.7.15)
```

Comment:
Could you please elaborate on this?

Reply:
node-gyp complition requires python. i use a tool called pyenv, like venv but better.
to use different python version in each project.
Not relevant, only .gitignore.